### ISSUE TITLE
Populate instance audit node with audit filename

### ISSUE THREAD

**Issue by lognaturel** [getodk/collect#1207]
When a form is set to perform an audit, corresponding filled out forms (instances) should include the audit's filename as described [here](https://github.com/opendatakit/xforms-spec/issues/94#issuecomment-290172183).

Currently, the instance has the following:
```xml
<n0:meta xmlns:n0="http://openrosa.org/xforms">
    <n0:audit />
</n0:meta>
```

Instead it should have: 
```xml
<n0:meta xmlns:n0="http://openrosa.org/xforms">
    <n0:audit><!-- the filename, added by client just like for <upload> --></n0:audit>
</n0:meta>
```

@nap2000 is this something you've looked at yet? Any thoughts or questions about it?

**Comment by nap2000** [comment]
I looked briefly at this and it seemed to me that it would require changes to java rosa. Any suggestions on updating instance files outside of JR would be welcome. There is currently no mechanism for the device user to change the audit timer file name which is hard coded as audit.csv.

**Comment by lognaturel** [comment]
This should be analogous to other binary files like images. If it needs to happen on the JR side, can you please file an issue there?

**Comment by nap2000** [comment]
OK.  So the behaviour you want in Java Rosa is that if the form has a meta value of <n0:audit /> then the content of that element should be set to "audit.csv"?

**Comment by lognaturel** [comment]
Right, and to be a little bit more precise, the form declaration will include an `audit` node in the `http://openrosa.org/xforms` namespace with whatever prefix used (typically `orx`). 

I haven't investigated how namespaces in filled forms are supposed to work. What is currently being generated with the namespace declared in the parent node is legal XML so I believe it's fine.

**Comment by nap2000** [comment]
Issue raised on JavaRosa https://github.com/opendatakit/javarosa/issues/76

**Comment by lognaturel** [comment]
Closed by #1384
